v2/internal/plugins/collector/proxy/context.go

```diff
@@ -94,6 +94,10 @@ func (pc *PluginContext) AddMetric(ns string, v interface{}, modifiers ...plugin
 		return fmt.Errorf("invalid value type (%T) for metric: %s", v, ns)
 	}
 
+	if pc.ctxManager.globalPrefix.enabled {
+		ns = fmt.Sprintf("/%s%s", pc.ctxManager.globalPrefix.name, ns)
```

Comment:
```suggestion
		ns = fmt.Sprintf("%s%s", pc.ctxManager.globalPrefix.name, ns)
```

Please change the logic so that the given prefix will include the separator as it not necessarily will be a `"/"` (and often it's not)

Reply:
changed